pkg/component/feature/feature.go

```diff
@@ -344,3 +313,20 @@ func renderComponents(cs []component.IstioComponent) (manifests name.ManifestMap
 	}
 	return
 }
+
+// buildCommonFeatureFields is an internal function to build the Common Feature Fields for specified feature.
+func buildCommonFeatureFields(opts *Options, ftname name.FeatureName) *CommonFeatureFields {
+	cff := &CommonFeatureFields{
+		Options: *opts,
+	}
+	if opts != nil && opts.Translator != nil && opts.Translator.FeatureMaps != nil {
```

Comment:
it's better to rewrite this condition as as the logical opposite and return if true - this reduces indentation.

Reply:
done.